### ISSUE TITLE
Crash immediately after launch

### ISSUE THREAD

**Issue by Pentacoro** [nipkownix/re4_tweaks#77]
Black screen and it stops working. At first it happened just after the 4gb popup, so first thing I tried was do the 4gb patch manually, and it kept crashing just the same way except for the popup not showing anymore. Then I replaced the modded BIO4 folder with the original, and it launched just fine! The tweeks GUI works and I was able to test some settings. Apparently it has something to do with the modded BIO4 folder (I checked it's size and everything should be fine).

Specs: 
AMD A10 4C + 8G 
AMD Radeon RX 560 Series 4gb VRAM 
16.0 GB RAM 
Windows 10 64bit
Clean Steam Install 
Of course the game ran 10/10 before

[bio4.exe.20220203151435.log](https://github.com/nipkownix/re4_tweaks/files/7997176/bio4.exe.20220203151435.log)

**Comment by nipkownix** [comment]
Did you delete the original BIO4 folder before placing the new one from the HD project?

**Comment by Pentacoro** [comment]
Yes

**Comment by emoose** [comment]
Seems to be a crash in `XCOMPRESS::fast_decode_verbatim_block`, maybe one of the .lfs files is corrupt in some way. 

If you install something like QuickSFV I can make a file you can use to check your install with, will upload in a sec.

**Comment by Pentacoro** [comment]
I'm guessing my safest bet is to download everything again, but in any case let's try that while I do

**Comment by emoose** [comment]
[BIO4-HDProject-SFV.zip](https://github.com/nipkownix/re4_tweaks/files/7997214/BIO4-HDProject-SFV.zip)

If you install QuickSFV you can extract that next to your BIO4 folder and run it, and that should verify the install for you, might take 5-10 minutes to go through it though.

**Comment by Pentacoro** [comment]
Yep, I already installed it. Let's check that out. Ty for the trouble :)

**Comment by nipkownix** [comment]
@emoose I sent you an email, just a heads up in case it gets lost in your spam folder.

**Comment by Pentacoro** [comment]
So yeah, it found a good deal of hash mismatches. The console just closed with no report file left behind tho, so I'll just download the files again.

**Comment by nipkownix** [comment]
Hmm, I wonder if albert should've included an automatic installer with hash checking and all.

Not sure if he's open to re-release an updated package with such changes. I'll have to ask him later.

**Comment by emoose** [comment]
> @emoose I sent you an email, just a heads up in case it gets lost in your spam folder.

Replied 👍

**Comment by Pentacoro** [comment]
So I downloaded the entire mod again and this time it launched, but now I'm experiencing some other consistent crashes on numerous different circumstances. I could start documenting them all along with their crashdumps, but running the SFV again it looks like there are still some hash mismatches, so I'll have to assume that's the problem again. Idk, I'm downloading it from the torrent file so idk idk idk

**Comment by Pentacoro** [comment]
In any case, thank you guys a ton for the patience.
